src/components/StakingPage/StakeInfo/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactNode } from 'react'
+import images from '../../../images';
+import styles from './styles'
+type Props = {
+  totalStakedHakka?: string;
+  totalSHakkaObtained?: string;
+  farmingSHakka?: string;
+  sHakkaBalance?: string;
+}
+
+type InfoItemProps = {
+  icon: any;
+  title: ReactNode;
+  value?: string;
+}
+
+const InfoItem = ({ icon, title, value }: InfoItemProps) => (
+  <div style={styles.infoItemWrapper}>
+    <div style={styles.titleWrapper}>
+      <img style={styles.icon} src={icon} />
+      <span>{title}</span>
+    </div>
+    <span style={(!value || parseFloat(value) === 0) ? { color: 'rgba(37, 62, 71, 0.5)' } : {}}>{value || '0.00'}</span>
```

Comment:
could try just `!parseFloat(value)` to replace `!value || parseFloat(value) === 0`